setup.py

```diff
@@ -64,7 +64,7 @@ def read(*rnames):
 #          'ZODB3',
 #          'Zope2',
 #          'Acquisition',
-          'plone.app.form',
+#          'plone.app.form',
```

Comment:
@MatthewWilkes any reason for just commenting it out? If you are removing, remove it for good 🙂